examples/cellar/gen/http/storage/client/types.go

```diff
@@ -285,7 +285,7 @@ func NewShowNotFound(body *ShowNotFoundResponseBody) *storage.NotFound {
 	return v
 }
 
-// Validate runs the validations defined on ShowNotFoundResponseBody
+// Validate runs the validations defined on show_not_foundResponseBody
```

Comment:
That doesn't look right.... Is that solved in a upcoming PR? or maybe examples need to be regenerated?